test/testcne/acl_test.c

```diff
@@ -370,12 +373,13 @@ test_classify_buid(struct cne_acl_ctx *acx, const struct cne_acl_ipv4vlan_rule *
 static int
 test_classify(void)
 {
+    tst_info("%s(%s)", __func__, "Test scalar and SSE ACL lookup");
```

Comment:
Please move the tst_info() line to after the function declaration or above line 380.

Reply:
Fixed all of these being before variable declarations in this file. I don't believe other files had this same problem.